src/main/resources/data/po-lines/306251-1_pending_electronic.json

```diff
@@ -0,0 +1,105 @@
+{
+  "id": "13fb64dc-cdf1-11e8-a8d5-f2801f1b9fd1",
+  "edition": "First edition",
+  "checkinItems": false,
+  "instanceId": "38448f80-9b98-45a5-a9f6-a96dd180e8f5",
```

Comment:
Please remove instanceId, the createInventory flag is false

Reply:
Changed